pkgs/by-name/ca/capacities/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  fetchurl,
+  appimageTools,
+  makeWrapper,
+  imagemagick,
+  lib,
+  ...
+}:
+let
+  pname = "capacities";
+  version = "1.43.34";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://capacities-desktop-app.fra1.cdn.digitaloceanspaces.com/Capacities-1.43.34.AppImage";
+    sha256 = "sha256-2XQOog60plA8yJX/y+89Us0fTGB2BPiKnLyCS2AjrXI=";
+  };
+
+  appimageContents = appimageTools.extractType2 { inherit name src; };
+in
+appimageTools.wrapType2 rec {
```

Comment:
I dont think this need to be `rec`, since `name` is available in the parent scope